Merge the color palette

ChengYanJin opened the pull request:
**Component**:
color palette, components

**Description**:
- Merge some colors in color palette:

 textSecondary (#A8B5C1) & textTertiary (#A7B6C3) => keep text Secondary #A8B5C1
 info button color (#027AFF) & Secondary color is blue (#037AFF) => > Keep #037AFF which should be the color of the most important action button, and the selection feedback color.

- Update the README
- Fix some bugs: in darks mode some components can't properly display the text
- Update the discrepancy between our current component and new design( progressbar, breadcrumb)
![image](https://user-images.githubusercontent.com/18453133/82785184-59df2000-9e62-11ea-9b62-7a53b8f86975.png)


---

<!-- Declare one or more issues to close once this PR gets merged -->

Closes: #188 

<!-- If you want to refer to an issue while not closing it, use:

See: #ISSUE_NUMBER

-->

ChengYanJin committed:
ColorPalette: Merge some colors

- textSecondary (#A8B5C1) & textTertiary (#A7B6C3) => keep text Secondary #A8B5C1
- info button color (#027AFF) & Secondary color is blue (#037AFF) => > Keep #037AFF which should be the color of the most important action button, and the selection feedback color.

Refs: #188

ChengYanJin committed:
text: Fix the bugs of text display issue in dark mode

Refs: #188

ChengYanJin committed:
test: Update the test

Refs: #188

bert-e commented:
# Hello chengyanjin,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/core-ui/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @rachedbenmustapha

* @nicolas2bert

* @carlito-scality

* @chengyanjin

* @monpote

ChengYanJin committed:
Sidebar: Set the default background color to `background`

Refs: #188

ChengYanJin committed:
Breadcrumb: Remove the padding in breadcrumb

Refs: #188

ChengYanJin committed:
Navbar: Update the default height

Refs: #188

ChengYanJin committed:
progressbar: Update with new design

- Add buildin label to specify the percentage
- Add backgroundColor to specify the unfilled color

Refs: #188

ChengYanJin commented:
/approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @rachedbenmustapha

* @nicolas2bert

* @carlito-scality

* @chengyanjin

* @monpote










*The following options are set:* **approve**

MonPote reviewed (approved):
LGTM

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/1.0`




There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/1.0`




Please check the status of the associated issue None.

Goodbye chengyanjin.
